frontend/src/components/common/form/ParentSelect.tsx

```diff
@@ -50,6 +52,7 @@ export const ParentSelect: React.FC<IParentSelect> = ({
   label,
   onChange,
   selectClosest,
+  convertValue = value => value,
```

Comment:
Is this doing anything?

Reply:
I think I'm missing something here.

Reply:
maybe we need a generic here to type the underlying object?

Reply:
I added an optional callback to transform the value before update formik value, this line is just defaulting to the raw value form the input.